### PR TITLE
edit/update team

### PR DESCRIPTION
**Pull request by dogiadat**

### CODE REVIEW

**Comment by nguyenluc312** [comment]
ook

File: app/controllers/admin/teams_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Admin::TeamsController < ApplicationController
-  before_action :load_countries, only: [:new, :create]
-  before_action :load_team, only: :show
+  before_action :load_countries, only: [:new, :create, :edit, :update]
```
Comment (phanbt58) [inline app/controllers/admin/teams_controller.rb]: except

Comment (huongnguyenmta) [inline app/controllers/admin/teams_controller.rb]: using except nhé :)

**Comment by huongnguyenmta** [comment]
OK

**Pull request merged**